models/golang/v3.0/gbfs/gbfs.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2024 MobilityData
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file was generated from JSON Schema using quicktype, do not modify it directly.
```

Comment:
Question: If this file (and other files) is generated, why commit it? Why not regenerate it as part of the build process? That's what is done with the java bindings, no?

Reply:
Understood. Thx